### PR TITLE
Issue#47-Apply-material-ui

### PR DESCRIPTION
**Pull request by chickengak**
#47 

Apply material-ui/core, material-icons

### CODE REVIEW

**Comment by LChoiSH** [comment]
짤ㅡ 짤ㅡ

File: src/frontend/src/App.js [hunk 1]
```diff
@@ -152,17 +183,44 @@ class App extends React.Component {
...
+      <Table className={classes.table}>
+        
+        <TableBody height = '100%'>
+       
+            <TableRow >
+              <TableCell component="th" scope="row" className= {classes.info}>
+              <App></App>
+              </TableCell>
+              <TableCell  className = {classes.map}>Map</TableCell>
+
```
Comment (Jelldo) [inline src/frontend/src/App.js]: same here

File: src/frontend/src/App.js [hunk 2]
```diff
@@ -152,17 +183,44 @@ class App extends React.Component {
...
-            Find loc
-          </button>
+          <IconButton color = 'primary' onClick={this.makeuser} aria-label="Make users">
+            <AddBoxIcon />
+          </IconButton> 
+          <IconButton color = "secondary" onClick={this.senddata} aria-label="Send data">
+            <SearchIcon />
+          </IconButton>
         </div>
       </div>
```
Comment (Jelldo) [inline src/frontend/src/App.js]: remove redundant space

File: src/frontend/src/App.js (hunk 1 above)
Comment (Jelldo) [reply src/frontend/src/App.js]: merge when resolve all comments

File: src/frontend/src/App.js [hunk 3]
```diff
@@ -1,7 +1,38 @@
...
+    height: '100%',
+  },
+  info: {
+    width: 300,
+    verticalAlign: 'top',
+  },
+  map: {
+    minWidth: 300,
+    backgroundColor: '#CEF6D8',
+
```
Comment (Jelldo) [inline src/frontend/src/App.js]: redundant space

**Pull request merged**